streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorAdapter.java

```diff
@@ -43,8 +44,24 @@ public static <KIn, VIn, KOut, VOut> Processor<KIn, VIn, KOut, VOut> adaptRaw(fi
         }
     }
 
+    @SuppressWarnings({"rawtypes", "unchecked"})
+    public static <KIn, VIn, KOut, VOut> Processor<KIn, VIn, KOut, VOut> adaptRaw(final Processor delegate) {
```

Comment:
Why do we need this? If we have a `api.Processor` at hand, there should be no need for wrapping it?